.cursor/BUGBOT.md

```
@@ -0,0 +1,155 @@
+## BugBot Rules (Index)
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)